db/schema.rb

```diff
@@ -30,9 +30,16 @@
     t.integer "source_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "access"
     t.index "url, md5((title)::text), md5(summary)", name: "index_news_on_url_and_title_and_summary", unique: true
   end
 
+  create_table "news_counters", force: :cascade do |t|
+    t.bigint "acessos"
```

Comment:
Vou repetir o que eu falei sobre consistência. Se puder deixar em inglês como o resto das outras coisas, acho que ia ser massa (até porque está escrito `access` na linha 33 ali em cima). Temos algumas opções:
```suggestion
    t.bigint "acess_count"
```